.travis.yml

```diff
@@ -0,0 +1,33 @@
+
+language: cpp
+
+addons:
+  apt:
+    sources:
+     - ubuntu-toolchain-r-test
+     - llvm-toolchain-trusty
+    packages:
+     - ninja-build
+     - llvm-dev
+     - libclang-dev
+     - clang
+
+before_install:
+ # Install a supported cmake version (>= 3.4.3)
+ - wget -O cmake.sh https://cmake.org/files/v3.10/cmake-3.10.0-rc1-Linux-x86_64.sh 
+ - sudo sh cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
+
+ # Extract the version number from the most-recently installed LLVM
+ - VERSION=`ls -t /usr/lib/ | grep '^llvm\-' | head -n 1 | sed 's/llvm\-\(.\+\)/\1/'`
+
+ - ROOT_PATH=`llvm-config-$VERSION --prefix`
+ - BIN_PATH=`llvm-config-$VERSION --bindir`
+
+script:
+# Build IWYU
+ - mkdir build && cd build && cmake -GNinja -DIWYU_LLVM_ROOT_PATH=$ROOT_PATH -DCMAKE_C_COMPILER=$BIN_PATH/clang -DCMAKE_CXX_COMPILER=$BIN_PATH/clang++ -DCMAKE_INSTALL_PREFIX=./ ../

```

Comment:
Could you split this line into the three steps (mkdir, cd, cmake) instead? Or does the one-liner add something?

Reply:
Each line in the script block will be executed even if the previous failed. So, it adds nothing really for *mkdir* and *cd* since they can't fail, but for *cmake* and *ninja* I think it makes sense. I'll change it as you suggested